Content.Client/GameObjects/Components/Mobs/BuckleVisualizer2D.cs

```diff
@@ -1,13 +1,18 @@
 ﻿using Content.Shared.GameObjects.Components.Mobs;
 using Content.Shared.GameObjects.Components.Strap;
 using JetBrains.Annotations;
+using Robust.Client.Animations;
 using Robust.Client.GameObjects;
+using Robust.Client.GameObjects.Components.Animations;
+using Robust.Client.Interfaces.GameObjects.Components;
+using Robust.Shared.Animations;
 using Robust.Shared.Maths;
+using System;
 
 namespace Content.Client.GameObjects.Components.Mobs
 {
-    [UsedImplicitly]
-    public class BuckleVisualizer2D : SpeciesVisualizer2D
+
+    public class BuckleVisualizer2D : AppearanceVisualizer
```

Comment:
At this rate just call this class something else, and drop the 2D part since we're not supporting 3D.
~~I propose calling it `SpessmanHorizontalVisualizer`~~